web/src/scheduled_messages_popover.js

```diff
@@ -165,6 +166,7 @@ export function initialize() {
                 focus_first_send_later_popover_item();
                 send_later_popover_keyboard_toggle = false;
             }
+            drafts.update_compose_draft_count();
```

Comment:
I think this function needs a comment for why this is here; I think I know why but it's not obvious from the name of the function. Maybe something like:

> // Make sure the compose drafts count, which is also displayed in this popover, has a current value.